staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/values_test.go

```diff
@@ -619,3 +619,67 @@ func TestMapper(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkUnstructuredToVal(b *testing.B) {
```

Comment:
Where was the before/after of these benchmarks for the optimization? (PR description is a good place to record these)